core/layout.go

```diff
@@ -294,13 +294,13 @@ func (ls *geomState) contentRangeDim(d math32.Dims) (cmin, cmax float32) {
 // totalRect returns Pos.Total -- Size.Actual.Total
 // as an image.Rectangle, e.g., for bounding box
 func (ls *geomState) totalRect() image.Rectangle {
-	return math32.RectFromPosSizeMax(ls.Pos.Total, ls.Size.Actual.Total)
```

Comment:
We had the previous behavior for a reason; I think it might have been related to border or shadow rendering or something. See c3ca7fa41d79826f21612532fa7524432e35bee8; you must test very thoroughly.

Reply:
Actually this is the wrong PR; see #1389.